test/unit/processing/siti-agri/transform-header.test.js

```diff
@@ -144,6 +145,13 @@ describe('Transform header', () => {
     expect(result.value).toBe(undefined)
   })
 
+  test('for CS return Payment Type', async () => {
+    const filename = 'SITICS0001_AP_20230315084313836.dat'
+    const headerData = ['H', 'CS000000001', '001', 'A0000001', '1', '1000000001', 'GBP', 'abc', 'NE00', 'GBP']
```

Comment:
What does the `abc` represent?

Reply:
Swapped out the value of `100` to be `abc` to test for when value is `NaN`but not required for this test so is a copy paste fail. Will update.